lib/Boolean.js

```diff
@@ -70,7 +77,7 @@
     Boolean.prototype.drawBackground = function(ctx, dimentions) {
       var statusWidth;
       Boolean.__super__.drawBackground.call(this, ctx, dimentions);
-      ctx.fillStyle = this.getStatus() ? config.get('color-success') : config.get('color-failure');
+      ctx.fillStyle = this.getStatus() == null ? config.get('color-unknown') : (this.getStatus() ? config.get('color-success') : config.get('color-failure'));
```

Comment:
I'm not a fan of nested ternary conditions. This would look better expanded in an actual if/else-if/else.